src/assets/scss/createItemPage.scss

```diff
@@ -60,7 +67,7 @@
     }
   }
 
-  button.md-btn {
+  button.iota-theme.md-btn {
```

Comment:
without this class all the global CSS is poluted and impossible to have your own Btn Style, I think we should also use something like BEM we can discuss that later if you want